### DISCUSSION TITLE
Favorite/Best downgrade method?

### DISCUSSION THREAD

**Discussion by KandyKaneMarvel** [3kh0/ext-remover#1063] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm highly considering downgrading considering the lack of good games and websites recently. I'm likely going to downgrade to version *118 so I can use Skiovox.

What method should I use? I don't have a PC, but I can probably use my phone as a substitute.

**Comment by ohiobombs** [comment, marked as answer]
just use a usb to download a recovery image and follow the instructions on the downgrading page
